zerver/lib/retention.py

```diff
@@ -22,11 +24,17 @@
         'table_name': 'zerver_submessage',
         'archive_table_name': 'zerver_archivedsubmessage'
     },
+    {
+        'class': UserMessage,
+        'archive_class': ArchivedUserMessage,
+        'table_name': 'zerver_usermessage',
+        'archive_table_name': 'zerver_archivedusermessage'
+    },
 ]  # type: List[Dict[str, Any]]
 
 @transaction.atomic
-def move_expired_rows(src_model: Any, raw_query: str, returning_id: bool=False,
-                      **kwargs: Any) -> List[int]:
+def move_rows(src_model: Any, raw_query: str, returning_id: bool=False,
+              **kwargs: Any) -> List[int]:
```

Comment:
Can you split the `move_rows` rename into its own commit?  I think that's be more readable.

Reply:
Done!